platforms/web/test-e2e/tests/account_test.ts

```diff
@@ -14,15 +14,16 @@ const consentCheckbox = 'Yes, I want to receive Blender updates by email';
 const firstName = 'John Q.';
 const lastName = 'Tester';
 
+Feature(`account`).retry(Number(process.env.TEST_RETRY_COUNT) || 0);
+
 runTestSuite(testConfigs.jwpSvod, 'JW Player', 'direct', false);
 runTestSuite(testConfigs.svod, 'Cleeng', 'resetLink', true);
 
 function runTestSuite(config: typeof testConfigs.svod, providerName: string, resetPasswordType: string, canEditEmail: boolean) {
   let loginContext: LoginContext;
 
-  Feature(`account - ${providerName}`).retry(Number(process.env.TEST_RETRY_COUNT) || 0);
-
-  Before(async ({ I }) => {
+  async function beforeScenario(I: CodeceptJS.I) {
```

Comment:
Does it not work to just move the `Before` to the top level where you moved `Feature`? This seems like an otherwise nice, simple enough fix, but I wish we didn't need to copy all the `await beforeScenario` calls. Seems too easy to forget to add when you create a new Scenario.

Reply:
Oh I see, Before depends on the runTestSuite args...

Reply:
I don't like the `beforeScenario` much either, we could also make this a custom step: `I.amLoggedIn(config);`

We also might be able to refactor the `accountContext` to a [shared value](https://codecept.io/parallel/#sharing-data-between-workers). We can then remove that from each test as well.